### ISSUE TITLE
Complete support for this

### ISSUE THREAD

**Issue by jjhesk** [timmutton/redex-plugin#1]
Are there anyways that u can prebuild or wrap the redex package inside this redex plugins?  Because we are using windows... many thanks..

**Comment by meoyawn** [comment]
I guess this is tracked in https://github.com/facebook/redex/issues/18

**Comment by timmutton** [comment]
Sorry for not replying to this sooner, my email filter must have hidden your issue. Installing Redex and all its dependencies is quite a complex task (especially for multiple platforms) which I unfortunately dont have the time for. And its probably beyond what the plugin should be doing anyway

**Comment by justinjhendrick** [comment]
Though we (redex) still don't support windows (yet), we are planning on putting redex binaries on maven (or similar) soon. We also plan for this gradle plugin to pull down a redex binary from the repository. This will avoid the need to build from source for most users.

**Comment by justinjhendrick** [comment]
Started the downloading work on this branch https://github.com/justinjhendrick/redex-plugin/tree/download_redex. It's a work in progress, but early comments are welcome.

**Comment by timmutton** [comment]
@justinjhendrick I may be reading this wrong, but your branch downloads the latest redex to the build dir of this project, it doesnt actually download it to the build dir of the project thats using this plugin

**Comment by justinjhendrick** [comment]
@timmutton, you're right... How do I put it in the project using the plugin's build dir?

**Comment by timmutton** [comment]
I don't know that you can include an executable in a jar. You might have to 
rework it so that the plugin does the download

Sent with AquaMail for Android
http://www.aqua-mail.com
